charts/azuremonitor-containers/templates/ama-logs-arc-k8s-crd.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or ( contains "microsoft.kubernetes/connectedclusters" (.Values.Azure.Cluster.ResourceId | lower)) ( contains "microsoft.hybridcontainerservice/provisionedclusters" (.Values.Azure.Cluster.ResourceId | lower)) }}
+{{- if or ( contains "microsoft.kubernetes/connectedclusters" (.Values.Azure.Cluster.ResourceId | lower) ) ( contains "microsoft.kubernetes/connectedclusters" (.Values.amalogs.env.clusterId | lower)) }}
```

Comment:
There is merge issue here. because of the merge issue, "microsoft.kubernetes/connectedclusters" check has been duplicated and "microsoft.hybridcontainerservice/provisionedclusters" check has been lost.